<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Win_Canshu</class>
 <widget class="QMainWindow" name="Win_Canshu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Wg_canshu" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>941</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>650</y>
      <width>951</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
border-image: url(:/new/home/home/info.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>111</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(178, 178, 178);
</string>
         </property>
         <property name="text">
          <string>Version:X.X.XX</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="MainButton" name="bt_jiaozhen">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>42</height>
        </size>
       </property>
       <property name="text">
        <string>扫描仪校正</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MainButton" name="pushButton_16">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>42</height>
        </size>
       </property>
       <property name="text">
        <string> 退出</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/new/home/home/tuichu.png</normaloff>:/new/home/home/tuichu.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>143</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>52</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>52</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/home/home/logo1.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(29, 63, 255);</string>
       </property>
       <property name="text">
        <string>参数 &gt; 参数</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="bt_canshu">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>71</y>
      <width>55</width>
      <height>146</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>146</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>146</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_weihu">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>223</y>
      <width>55</width>
      <height>146</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>146</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>146</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Wg_WeiHu" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>941</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainButton</class>
   <extends>QPushButton</extends>
   <header location="global">mainbutton.h</header>
  </customwidget>
  <customwidget>
   <class>Wg_canshu</class>
   <extends>QWidget</extends>
   <header location="global">wg_canshu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Wg_WeiHu</class>
   <extends>QWidget</extends>
   <header>wg_weihu.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
